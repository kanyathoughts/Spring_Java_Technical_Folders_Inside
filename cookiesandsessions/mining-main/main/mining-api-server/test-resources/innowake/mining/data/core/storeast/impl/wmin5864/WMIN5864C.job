//T446KADL JOB (00200091,N591265),'RTS DICKSO BDG6 501H',
//  MSGLEVEL=(1,1),TIME=99,CLASS=H,MSGCLASS=R
//*JOBPARM L=100,T=90
//*
//STEP03    	EXEC PGM=ADARUN
//*
//CNDSAMP JOB CLASS=6,NOTIFY=&SYSUID,COND=((10,LT,STP01),(10,GT,STP02))
//STP02    EXEC PGM=IEBGENER
