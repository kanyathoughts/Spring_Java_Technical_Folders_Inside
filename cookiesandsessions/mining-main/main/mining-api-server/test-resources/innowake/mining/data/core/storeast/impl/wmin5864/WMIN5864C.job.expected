AstNode [type=JCLRootNode, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1, hasAstModuleId=1]
 AstNode [type=JclJobNode, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1, properties=[JOB_NAME=T446KADL]]
  AstNode [type=JclStepExec, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1, properties=[is_returning=true, MODULE_NAME=ADARUN, qualifiedStepName=WMIN5864C.STEP03.EXEC_PGM]]
 AstNode [type=JclJobNode, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1, properties=[JOB_NAME=CNDSAMP]]
  AstNode [type=JclCondition, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1]
  AstNode [type=JclStepExec, moduleId=1, location=AstNodeLocation [retracedOffset=-1, retracedLength=1, assembledOffset=-1, assembledLength=1, rootRelativeOffset=-1, rootRelativeLength=1, rootRelativeStartLineNumber=0, rootRelativeEndLineNumber=1], inclusionLevel=-1, inclusionOrdinal=-1, properties=[is_returning=true, MODULE_NAME=IEBGENER, qualifiedStepName=WMIN5864C.STP02.EXEC_PGM]]
